endpoints/openrtb2/sample-requests/amp/consent-through-query/addtl-consent-through-query.json

```diff
@@ -63,11 +71,44 @@
     ],
     "user": {
       "ext": {
-        "ConsentedProvidersSettings":{"consented_providers":"1~1.35.41.101"},
-        "consented_providers_settings":{"consented_providers":[1,35,41,101]}
+        "ConsentedProvidersSettings": {
+          "consented_providers": "1~1.35.41.101"
+        },
+        "consented_providers_settings": {
+          "consented_providers": [
+            1,
+            35,
+            41,
+            101
+          ]
+        }
       }
     },
-    "ext": {"prebid":{"cache":{"bids":{"returnCreative":null},"vastxml":null},"channel":{"name":"amp","version":""},"targeting":{"pricegranularity":{"precision":2,"ranges":[{"min":0,"max":20,"increment":0.1}]},"includewinners":true,"includebidderkeys":true,"includebrandcategory":null,"includeformat":false,"durationrangesec":null,"preferdeals":false}}}
+    "ext": {
+      "prebid": {
+        "cache": {
+          "bids": {}
+        },
+        "channel": {
+          "name": "amp",
+          "version": ""
+        },
+        "targeting": {
+          "pricegranularity": {
+            "precision": 2,
+            "ranges": [
+              {
+                "min": 0,
+                "max": 20,
+                "increment": 0.1
+              }
+            ]
+          },
+          "includewinners": true,
+          "includebidderkeys": true
```

Comment:
Changes due to auto-formatting and removal of omitempty fields. Same goes for most of the json test files.